test/gst-msdk/transcode/transcoder.py

```diff
@@ -11,44 +11,171 @@
 @slash.requires(have_gst)
 @slash.requires(*have_gst_element("msdk"))
 @slash.requires(*have_gst_element("checksumsink2"))
```

Comment:
Please add `@slash.requires(using_compatible_driver)`

Reply:
added it